### DISCUSSION TITLE
Collection name is sometimes created with lower/camel case when using Studio?

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#16017] [Q&A] [answered]
When I create a new document from the studio using the following data the collection name becomes lower case.

```json
{
    "Name": "...",
    "@metadata": {
        "@collection": "Homes",
        "Raven-Clr-Type": "Wieldy.Web.Models.Home, Wieldy.Web"
    }
}
```

https://user-images.githubusercontent.com/319720/225242331-8f2b8f67-495b-41f5-afae-5b0febf305f1.mp4

If I use the following data the collection name doesn't change.

```json
{
    "Name": "...",
    "@metadata": {
        "@collection": "StartPage",
        "Raven-Clr-Type": "Wieldy.Web.Models.Home, Wieldy.Web"
    }
}
```

https://user-images.githubusercontent.com/319720/225242460-2b8f4f63-e498-4669-b839-e7e184928cc7.mp4

This is pretty annoying because the casing of the collection seems to be important in indexes, subscriptions and so on.

I'm using the following versions in your cloud.

**Server version**
5.4.101-cloud-54034
**Studio version**
5.4.101-cloud-54034

**Comment by garayx** [comment]
hi

after creating the document in studio (when you see `homes` (lowercase)), can you press on the json icon, and share the raw json here ? 

Do you still have lowercase `homes` there ?

![image](https://user-images.githubusercontent.com/16625387/225294388-0bd06d16-73e4-42eb-8d07-6bdc242613a7.png)

**Comment by ayende** [comment, marked as answer]
Collection names in RavenDB are case insensitive.

The _first_ document that you create in a collection determine the standard casing for it.

What you have done probably is:

* Created a collection with `homes`
* Deleted the collection
* Created a document with `Homes`

The first first document after this, looks like it will use the initial collection naming.
This doesn't matter the second time around.

**Reply by marcuslindblom** [reply]
Thats exactly what I did and how I can reproduce it.
